src/styles/index.css

```diff
@@ -0,0 +1,7 @@
+@import "./reset.css";
+
+@import "./variables.css";
+
+@import "./cards.css";
+
+@import "./stars.css";
```

Comment:
I would recommend creating a brand new directory for blocks: 'styles/blocks/stars.css'